src/workers/jobs.js

```diff
@@ -158,10 +158,12 @@ export async function uploadContacts(job) {
     await CampaignContact.save(savePortion)
   }
 
+  const optOutsByOrgId = function() { this.select('cell').from('opt_out').where('organization_id', campaign.organization_id) }
```

Comment:
since this is the same code, as below, can we abstract it out to a global file function or constant?